src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -140,7 +148,7 @@ export class CdkMenuItemTrigger implements OnDestroy {
    */
   _toggleOnMouseEnter() {
     const menuStack = this._getMenuStack();
-    if (!menuStack.isEmpty() && !this.isMenuOpen()) {
+    if (!menuStack?.isEmpty() && !this.isMenuOpen()) {
```

Comment:
The first part of this expression is a little hard to parse. Is there a way to make it clearer?